site/src/components/NavigationBar/components/Link.tsx

```diff
@@ -34,13 +34,13 @@ class Link extends React.PureComponent<LinkProps> {
     } = this.props
 
     return (
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
What is this one taking care of?

Reply:
This old component has an `onClick` prop that gets used on a link, which is a no-no. What makes it tricky to fix is that it also has an `href` prop, so it can't just be blanket changed to a `button`. I'd have to switch the element based on what was passed in, and I don't think it's worth the effort for this legacy stuff.